pypesto/sample/parallel_tempering.py

```diff
@@ -261,9 +264,17 @@ def compute_log_evidence(
                 y=mean_loglike_per_beta,
                 x=temps,
             )
+        elif method == "steppingstone":
+            ss_log_evidences = np.zeros(len(temps) - 1)
```

Comment:
Maybe nicer to put this into a separate function as done for the trapezoidal and the simpson above.

Reply:
Additionally, a source for the formula would then be great.

Reply:
Another point here. In the paper you refer to in the example notebook, they factor out the largest  log-likelihood to improve numerical stability. I don't see this to be done explicitly in your implementation. Any reason for that or is is just a bit more hidden and I don't see it directly?

Reply:
It is kind of hidden int the `logsumexp` from `scipy`. I used the maximum likelihood times the temperature difference to stabilize the logsumexp, while the original formulation uses only the maximum likelihood, but this is equivalent.